src/openassetio-core/tests/CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ target_link_libraries(
 # Create CTest target
 
 # Requires: openassetio.internal.install
+# (Disable line too long as it makes this command awkward.)
+# cmake-lint: disable=C0301
```

Comment:
[question] Can we disable linter checks using a more human-readable tag?  With other linters you can disable by either the code or a string tag.

Reply:
I don't think so, it's why I've put the additional comment string above. I did try to put the english language description as the disable arg, but to no effect.

The only disable docs i can find are on using the code : https://cmake-format.readthedocs.io/en/latest/lint-inline.html

Reply:
One thing to be aware of, based on the docs, that `disable` applies for the whole (rest of the?) file, i.e. from the docs you've linked:

> must be specified at body scope [...] and it applies for the duration of the scope

So we should use sparingly, at least until it's made more configurable:

> behavior will likely change in the future

Reply:
Aye, point well taken.

(Awkwardly, thanks cmake,) Wrapped these lines, rather than just disabling them.

https://github.com/OpenAssetIO/OpenAssetIO/pull/811/commits/d5be5c5aa62255eb7a8c54f4e686868ffbf3bb3e